routers/main.js

```diff
@@ -4,15 +4,15 @@ import passport from 'passport';
 import GithubStrategy from 'passport-github';
 import {SuggestionsController} from '../controllers/suggestionsController';
 const Strategy = GithubStrategy.Strategy;
-import {TestsService} from '../services/testsService';
-import moment from 'moment';
 
 import config from '../ludwig-conf.js';
 
-const testsService = new TestsService();
-
 import {HistoryController} from '../controllers/historyController';
+
 const historyController = new HistoryController(config);
+import {ListTestsController} from '../controllers/listTestsController';
+const listTestsController = new ListTestsController();
```

Comment:
File-wide instantiations make it pretty clear there's something wrong with making all of those classes.